lib/install.ps1

```diff
@@ -86,9 +86,13 @@ function Find-Manifest($app, $bucket) {
         $manifest = url_manifest $url
     } else {
         # check buckets
-        $manifest, $bucket = find_manifest $app $bucket
+        if (!$bucket) {
```

Comment:
In general, when you are using both branches of `if` statement, then it is not suitable to negate the result.

```suggestion
        if ($bucket) {
            $manifest, $null = find_manifest $app $bucket
        } else {
            $manifest, $bucket = find_manifest $app
        }
```